guillotina/tests/fixtures.py

```diff
@@ -638,12 +638,9 @@ def redis_container():
 
 @pytest.fixture(scope="session")
 def memcached_container():
-    import pytest_docker_fixtures
-
-    host, port = pytest_docker_fixtures.memcached_image.run()
+    host, port = os.environ["MEMCACHED"].split(":")
```

Comment:
If env var is not defined what happens?

Reply:
yup, right. Added back using pytest docker fixtures if env var is not configured